core/flyout_base.js

```diff
@@ -46,6 +46,7 @@ Blockly.Flyout = function(workspaceOptions) {
    */
   this.workspace_ = new Blockly.WorkspaceSvg(workspaceOptions);
   this.workspace_.isFlyout = true;
+  this.workspace_.setVisible(false);
```

Comment:
I think this is to keep the flyout from rerendering on theme changes right? If so please correct me. I might consider leaving a comment, but its your call.

Reply:
Yes, and more importantly so the flyout representation of visibility is reflected in the workspace. https://github.com/google/blockly/blob/master/core/flyout_base.js#L130 is set to false